### PR TITLE
feat: Add extending warp route page

### PR DESCRIPTION
**Pull request by ltyu**
url is: https://hyp-v3-docs-git-ltyu-warp-apply-abacus-works.vercel.app/docs/guides/extending-warp-route

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const sidebars = {
     {
       type: "doc",
       id: "intro",
-      label: "1) Choose Hyperlane",
+      label: "1 - Choose Hyperlane",
```
Comment (ltyu) [inline sidebars.js:23]: Todo: remove numbering all together

File: docs/guides/extending-warp-route.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+# How to extend an existing Warp Route
+
+# Using the Hyperlane CLI
+
+After deploying your warp route, you may want to extend it to a new chain. This happens in 2 steps: new contract get deployed to the new chain, and all existing routers get enrolled with the new one.
+
+You can extend an existing warp route using the [Hyperlane CLI](https://docs.hyperlane.xyz/docs/reference/cli).
+<PrerequisitesPartial />
+:::warning
+Gnosis Safe Users: The private key that owns the warp route must either be a Safe signer or [delegate](https://docs.safe.global/core-api/transaction-service-guides/delegates) to be able to propose update transactions.
```
Comment (Mo-Hussain) [inline docs/guides/extending-warp-route.mdx:18]: Would EOA be a better description than private key?

Comment (ltyu) [reply docs/guides/extending-warp-route.mdx:18]: Hm maybe

I don't feel strongly about either and as you can see I use it interchangeably. I could pick one to be consistent if there's not strong opinions about it

Comment (Mo-Hussain) [reply docs/guides/extending-warp-route.mdx:18]: It's a personal preference but I think EOA is more appropriate!

File: docs/guides/extending-warp-route.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+import WarpReadSymbolChainPartial from '/docs/partials/warp-routes/commands/_warp-read-symbol-chain.mdx'
+import WarpApplySymbolConfigDefaultPartial from '/docs/partials/warp-routes/commands/_warp-apply-symbol-config-default.mdx'
+import WarpApplySymbolConfigDefaultStrategyPartial from '/docs/partials/warp-routes/commands/_warp-apply-symbol-config-default-strategy.mdx'
+import WarpReadSymbolSelectPartial from '/docs/partials/warp-routes/_warp-read-symbol-selection.mdx'
+
+# How to extend an existing Warp Route
+
+# Using the Hyperlane CLI
+
+After deploying your warp route, you may want to extend it to a new chain. This happens in 2 steps: new contract get deployed to the new chain, and all existing routers get enrolled with the new one.
```
Comment (jmrossy) [inline docs/guides/extending-warp-route.mdx:13]: typo 'new contract get'

File: docs/guides/extending-warp-route.mdx (hunk 2 above)
Comment (jmrossy) [inline docs/guides/extending-warp-route.mdx:18]: Is this warning unique to extending warp routes or for all safe-based CLI txs?

File: docs/guides/extending-warp-route.mdx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+You can extend an existing warp route using the [Hyperlane CLI](https://docs.hyperlane.xyz/docs/reference/cli).
+<PrerequisitesPartial />
+:::warning
+Gnosis Safe Users: The private key that owns the warp route must either be a Safe signer or [delegate](https://docs.safe.global/core-api/transaction-service-guides/delegates) to be able to propose update transactions.
+:::
+
+If you already have the warp config, skip to **Step 1**.
```
Comment (jmrossy) [inline docs/guides/extending-warp-route.mdx:21]: Maybe link Step 1

File: docs/guides/extending-warp-route.mdx [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+To generate the warp route config using the Hyperlane CLI, locate your token symbol on the chain on which it is deployed:
+
+<WarpReadSymbolChainPartial />
+
+<WarpReadSymbolSelectPartial />
+
+You can then choose the warp route to extend.
+
+After running `hyperlane warp read`, you should see a similar config:
```
Comment (jmrossy) [inline docs/guides/extending-warp-route.mdx:31]:
```suggestion
After running `hyperlane warp read`, the CLI will display a config similar to the following
```

File: docs/guides/extending-warp-route.mdx [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+  symbol: "ETH"
+  decimals: 18
+  totalSupply: "0"
+  remoteRouters:
+    "84532": "0xB38cc797Ccc6D8763439a07CEa052AF253281de6"
+    "11155111": "0x4e63147e72d029Eb8b37F375B3837B2644622686"
+  type: "synthetic"
+```
+
+By completing these steps, you've successfully extended your Warp Route to a new chain.
```
Comment (jmrossy) [inline docs/guides/extending-warp-route.mdx:178]: You can cut this line, doesn't add anything :)

File: docs/guides/extending-warp-route.mdx (hunk 2 above)
Comment (ltyu) [reply docs/guides/extending-warp-route.mdx:18]: Right now this only applies to extending warp routes. In the future, i expect it to apply to others

File: docs/guides/extending-warp-route.mdx (hunk 5 above)
Comment (ltyu) [reply docs/guides/extending-warp-route.mdx:31]: Fixed

**Review by jmrossy — APPROVED** [review]

**Pull request merged**